### PR TITLE
ASCII()

### PR DESCRIPTION
**Pull request by rachithrr**

### CODE REVIEW

**Comment by rachithrr** [comment]
merge after https://github.com/molecula/featurebase/pull/2378/files

File: sql-preview/functions/_posts/2022-12-16-sql-ascii.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+_expr_
+The string to return the ASCII value for. The argument `expr` is any expression of type `string`.
+ 
+
+## Return Type
+`string`
+
+## Return Value
+`ascii()` returns the ASCII value. If the evaluated expression `expr` contains more than one character, returns the ascii value of the first character.
```
Comment (ljsinclair) [inline sql-preview/functions/_posts/2022-12-16-sql-ascii.md:24]: Am I right in thinking `ascii() returns the ascii value of the evaluated expression? (if so, need to make this clearer).

With the second sentence, I'm wondering what the benefit of `ascii()` actually is to an end-user. This should be made clearer in the opening paragraph.

File: sql-preview/functions/_posts/2022-12-16-sql-ascii.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+## Return Value
+`ascii()` returns the ASCII value. If the evaluated expression `expr` contains more than one character, returns the ascii value of the first character. 
+
+## Remarks
+None
+
+## Examples
+A. ASCII function on a column.
+
+```sql
```
Comment (ljsinclair) [inline sql-preview/functions/_posts/2022-12-16-sql-ascii.md:32]: What's the result of these SQL statements?

File: sql-preview/functions/_posts/2022-12-16-sql-ascii.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+create table segments
+    (_id id, segment string);
+
+insert into segments(_id, segment)
+    values (1,'red')
+
+select _id, ascii(segment) as segment from segments;
++-----+----------+
```
Comment (ljsinclair) [inline sql-preview/functions/_posts/2022-12-16-sql-ascii.md:40]: Or is this the result? In which case, need to make this clearer please :D

**Review by ljsinclair — CHANGES_REQUESTED** [review]
Hi there, some comments on this mainly around clarification :)

File: sql-preview/functions/_posts/2022-12-16-sql-ascii.md (hunk 3 above)
Comment (gthrone25) [reply sql-preview/functions/_posts/2022-12-16-sql-ascii.md:40]: @ljsinclair we previously approved a good amount of pages that already uses this format. I think it's fine, as select is the only statement here that returns a result set.

Comment (ljsinclair) [reply sql-preview/functions/_posts/2022-12-16-sql-ascii.md:40]: No worries, I'll fix when I get to the edits on the SQL pages

File: sql-preview/functions/_posts/2022-12-16-sql-ascii.md (hunk 1 above)
Comment (rachithrr) [reply sql-preview/functions/_posts/2022-12-16-sql-ascii.md:24]: Evaluated expression means that the argument can be any function that evaluates to a string. 
*With the new patch, restricting the length of string to 1 based on Pat's suggestion to keep it restricted.  We might add the ability to pass string of length > 1 in the future.

**Review by ljsinclair — APPROVED** [review]

**Pull request merged**